tests/test_shapes.py

```diff
@@ -0,0 +1,75 @@
+import sys
+from pdb import set_trace as bp
+import rdflib
+import brickschema
+from bricksrc.namespaces import A, OWL, RDFS, SKOS, BRICK, SH, BSH, bind_prefixes
+import pyshacl
+
+schema_g = brickschema.Graph().parse('shacl/BrickShape.ttl', format='turtle')
+#ont_g = brickschema.Graph().parse('Brick.ttl', format='turtle')
+bind_prefixes(schema_g)
+
+prefixes = """
+@prefix brick: <https://brickschema.org/schema/Brick#> .
+@prefix : <http://example.com#> .
+"""
+
+base_data = prefixes + """
+:equip a brick:Equipment.
+:point a brick:Point.
+:loc a brick:Location.
+"""
+
+def test_no_relations():
+    data = base_data
+    data_g = rdflib.Graph().parse(data=data, format='turtle')
+    conforms, results_graph, results_text = pyshacl.validate(
+        data_g, shacl_graph=schema_g,
+    )
+    assert conforms
+
+def test_equip():
+    valid_data = base_data + """
+:equip brick:hasLocation :loc.
+"""
+    valid_g = rdflib.Graph().parse(data=valid_data, format='turtle')
+    conforms, results_graph, results_text = pyshacl.validate(
+        valid_g, shacl_graph=schema_g
+    )
+    assert conforms
```

Comment:
Same here; you can simplify syntax with the brickschema Graph. I think we also want to throw away (or assign to `_`) the returned values we aren't using

Reply:
I was editing things, and then thought it's better not using brickschema package when it doesn't provide a large benefit. It's a unnecessary dependency while also causing circular dependencies.

Reply:
The dependency already exists and it isn't circular. I feel like it is better to keep the implementation of certain actions, such as validation, consistent across the codebase

Reply:
I meant the circular dependency in functionality of those two repositories in general. The python package helps to easily use the ontology, while we are using that package to validate the ontology. Anyway, we don't have to waste our energy here, and I can use brickschema package.

The package currently does not support loading from string data, which is used in this code. I'd appreciate if you can update the package to enable that. (another function called `load_data`?)

Reply:
The `brickschema.Graph` object is a subclass of `rdflib.Graph` so you can use the same methods here (e.g. `g.parse`)